2024-03-15 22:44:39,210 INFO    StreamThr :19352 [internal.py:wandb_internal():86] W&B internal server running at pid: 19352, started at: 2024-03-15 22:44:39.209033
2024-03-15 22:44:39,238 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status
2024-03-15 22:44:39,240 INFO    WriterThread:19352 [datastore.py:open_for_write():87] open: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\run-0w0d7rkn.wandb
2024-03-15 22:44:39,245 DEBUG   SenderThread:19352 [sender.py:send():378] send: header
2024-03-15 22:44:39,287 DEBUG   SenderThread:19352 [sender.py:send():378] send: run
2024-03-15 22:44:39,784 INFO    SenderThread:19352 [dir_watcher.py:__init__():211] watching files in: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files
2024-03-15 22:44:39,784 INFO    SenderThread:19352 [sender.py:_start_run_threads():1099] run started: 0w0d7rkn with start time 1710510279.210663
2024-03-15 22:44:39,806 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: check_version
2024-03-15 22:44:39,806 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: check_version
2024-03-15 22:44:39,874 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: run_start
2024-03-15 22:44:40,821 DEBUG   HandlerThread:19352 [system_info.py:__init__():26] System info init
2024-03-15 22:44:40,821 DEBUG   HandlerThread:19352 [system_info.py:__init__():41] System info init done
2024-03-15 22:44:40,821 INFO    HandlerThread:19352 [system_monitor.py:start():194] Starting system monitor
2024-03-15 22:44:40,821 INFO    SystemMonitor:19352 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-03-15 22:44:40,823 INFO    HandlerThread:19352 [system_monitor.py:probe():214] Collecting system info
2024-03-15 22:44:40,828 INFO    SystemMonitor:19352 [interfaces.py:start():190] Started cpu monitoring
2024-03-15 22:44:40,829 INFO    SystemMonitor:19352 [interfaces.py:start():190] Started disk monitoring
2024-03-15 22:44:40,830 INFO    SystemMonitor:19352 [interfaces.py:start():190] Started gpu monitoring
2024-03-15 22:44:40,832 INFO    SystemMonitor:19352 [interfaces.py:start():190] Started memory monitoring
2024-03-15 22:44:40,849 INFO    SystemMonitor:19352 [interfaces.py:start():190] Started network monitoring
2024-03-15 22:44:40,868 DEBUG   HandlerThread:19352 [system_info.py:probe():150] Probing system
2024-03-15 22:44:40,871 DEBUG   HandlerThread:19352 [system_info.py:_probe_git():135] Probing git
2024-03-15 22:44:40,871 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:44:40,871 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:44:40,989 DEBUG   HandlerThread:19352 [system_info.py:_probe_git():143] Probing git done
2024-03-15 22:44:40,989 DEBUG   HandlerThread:19352 [system_info.py:probe():198] Probing system done
2024-03-15 22:44:40,989 DEBUG   HandlerThread:19352 [system_monitor.py:probe():223] {'os': 'Windows-10-10.0.22631-SP0', 'python': '3.9.16', 'heartbeatAt': '2024-03-15T13:44:40.868564', 'startedAt': '2024-03-15T13:44:39.174156', 'docker': None, 'cuda': None, 'args': ('--use_wandb', 'True', '--experiment_name', 'use_step_weight'), 'state': 'running', 'program': 'C:\\Users\\USER\\Desktop\\mini-marl\\vdn\\main.py', 'codePathLocal': 'main.py', 'codePath': 'vdn\\main.py', 'git': {'remote': 'https://github.com/singforai/mini-marl.git', 'commit': 'a8a6b055b52b59211f15021a7195aad66f4d99f9'}, 'email': 'singfor7012@gmail.com', 'root': 'C:/Users/USER/Desktop/mini-marl', 'host': 'LAPTOP-FP8FOEQU', 'username': 'USER', 'executable': 'C:\\Users\\USER\\anaconda3\\envs\\torch\\python.exe', 'cpu_count': 4, 'cpu_count_logical': 8, 'cpu_freq': {'current': 2803.0, 'min': 0.0, 'max': 2803.0}, 'cpu_freq_per_core': [{'current': 2803.0, 'min': 0.0, 'max': 2803.0}], 'disk': {'/': {'total': 459.74316024780273, 'used': 411.53125}}, 'gpu': 'NVIDIA GeForce MX450', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce MX450', 'memory_total': 2147483648}], 'memory': {'total': 15.729598999023438}}
2024-03-15 22:44:40,989 INFO    HandlerThread:19352 [system_monitor.py:probe():224] Finished collecting system info
2024-03-15 22:44:40,989 INFO    HandlerThread:19352 [system_monitor.py:probe():227] Publishing system info
2024-03-15 22:44:40,990 DEBUG   HandlerThread:19352 [system_info.py:_save_conda():207] Saving list of conda packages installed into the current environment
2024-03-15 22:44:41,828 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\conda-environment.yaml
2024-03-15 22:44:42,918 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:44:42,919 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:44:42,919 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:44:42,919 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:44:44,972 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:44:44,972 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:44:44,973 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:44:44,973 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:44:46,885 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\conda-environment.yaml
2024-03-15 22:44:47,025 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:44:47,025 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:44:47,025 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:44:47,025 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:44:47,030 DEBUG   HandlerThread:19352 [system_info.py:_save_conda():219] Saving conda packages done
2024-03-15 22:44:47,033 INFO    HandlerThread:19352 [system_monitor.py:probe():229] Finished publishing system info
2024-03-15 22:44:47,040 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:44:47,040 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: keepalive
2024-03-15 22:44:47,041 DEBUG   SenderThread:19352 [sender.py:send():378] send: files
2024-03-15 22:44:47,041 INFO    SenderThread:19352 [sender.py:_save_file():1365] saving file wandb-metadata.json with policy now
2024-03-15 22:44:47,242 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: python_packages
2024-03-15 22:44:47,242 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: python_packages
2024-03-15 22:44:47,243 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: stop_status
2024-03-15 22:44:47,244 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-15 22:44:47,244 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: stop_status
2024-03-15 22:44:47,492 DEBUG   SenderThread:19352 [sender.py:send():378] send: telemetry
2024-03-15 22:44:47,740 INFO    wandb-upload_0:19352 [upload_job.py:push():131] Uploaded file C:\Users\USER\AppData\Local\Temp\tmpckhej3pbwandb\v8250hsx-wandb-metadata.json
2024-03-15 22:44:47,899 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\wandb-metadata.json
2024-03-15 22:44:47,900 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\requirements.txt
2024-03-15 22:44:47,900 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:44:49,084 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:44:49,084 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:44:49,084 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:44:49,084 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:44:49,917 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:44:50,052 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:44:51,137 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:44:51,137 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:44:51,137 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:44:51,137 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:44:51,947 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:44:53,197 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:44:53,197 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:44:53,198 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:44:53,198 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:44:53,976 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:44:55,147 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:44:55,248 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:44:55,248 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:44:55,248 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:44:55,248 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:44:55,993 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:44:57,330 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:44:57,331 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:44:57,331 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:44:57,331 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:44:58,006 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:44:59,366 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:44:59,401 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:44:59,401 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:44:59,401 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:00,027 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:00,167 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:45:01,482 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:01,482 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:01,482 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:01,482 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:02,045 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:02,249 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-15 22:45:02,250 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: stop_status
2024-03-15 22:45:02,250 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: stop_status
2024-03-15 22:45:03,531 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:03,531 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:03,532 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:03,532 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:04,058 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:05,192 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:45:05,617 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:05,617 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:05,617 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:05,617 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:06,075 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:07,692 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:07,692 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:07,692 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:07,692 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:08,094 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:09,749 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:09,750 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:09,750 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:09,750 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:10,104 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:10,235 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:45:11,111 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\config.yaml
2024-03-15 22:45:12,115 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:12,638 ERROR   gpu       :19352 [interfaces.py:aggregate():161] Failed to serialize metric: division by zero
2024-03-15 22:45:12,639 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:12,640 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:14,127 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:14,655 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:14,655 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:14,655 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:14,655 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:15,493 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:45:16,152 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:16,667 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:16,667 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:16,667 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:16,667 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:17,246 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: stop_status
2024-03-15 22:45:17,246 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: stop_status
2024-03-15 22:45:17,259 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-15 22:45:18,176 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:18,676 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:18,676 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:18,676 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:18,676 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:20,201 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:20,657 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:45:20,684 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:20,684 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:20,684 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:20,684 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:22,231 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:22,692 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:22,692 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:22,692 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:22,692 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:24,256 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:24,706 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:24,707 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:24,707 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:24,707 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:25,670 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:45:26,290 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:26,720 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:26,721 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:26,721 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:26,721 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:28,386 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:28,734 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:28,735 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:28,735 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:28,735 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:30,450 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:30,691 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:45:30,736 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:30,736 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:30,736 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:30,736 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:32,251 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: stop_status
2024-03-15 22:45:32,252 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: stop_status
2024-03-15 22:45:32,268 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-15 22:45:32,543 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:32,741 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:32,741 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:32,742 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:32,742 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:34,623 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:34,754 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:34,754 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:34,755 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:34,755 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:35,782 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:45:36,700 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:36,761 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:36,761 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:36,761 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:36,761 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:38,780 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:38,780 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:38,780 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:38,780 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:38,781 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:40,885 DEBUG   SystemMonitor:19352 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-03-15 22:45:40,887 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:40,887 DEBUG   SenderThread:19352 [sender.py:send():378] send: stats
2024-03-15 22:45:40,887 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:40,889 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:45:40,889 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:40,889 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:40,890 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:41,892 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:43,821 ERROR   gpu       :19352 [interfaces.py:aggregate():161] Failed to serialize metric: division by zero
2024-03-15 22:45:43,822 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:43,822 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:43,965 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:45,834 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:45,834 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:45,834 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:45,834 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:45,984 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:45:46,043 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:47,257 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-15 22:45:47,258 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: stop_status
2024-03-15 22:45:47,258 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: stop_status
2024-03-15 22:45:47,849 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:47,849 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:47,849 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:47,849 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:48,134 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:49,856 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:49,856 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:49,856 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:49,856 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:50,202 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:51,064 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:45:51,866 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:51,866 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:51,866 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:51,866 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:52,254 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:53,871 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:53,871 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:53,871 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:53,872 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:54,307 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:55,874 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:55,874 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:55,874 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:55,874 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:56,161 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:45:56,343 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:57,886 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:57,886 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:57,886 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:57,886 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:58,418 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:45:59,887 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:59,887 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:45:59,887 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:45:59,887 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:00,488 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:01,198 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:46:01,895 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:01,895 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:01,895 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:01,895 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:02,272 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: stop_status
2024-03-15 22:46:02,272 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-15 22:46:02,273 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: stop_status
2024-03-15 22:46:02,538 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:03,911 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:03,911 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:03,911 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:03,912 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:04,616 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:05,920 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:05,920 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:05,920 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:05,920 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:06,265 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:46:06,670 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:07,929 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:07,929 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:07,929 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:07,929 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:08,728 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:09,936 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:09,936 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:09,936 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:09,936 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:10,781 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:10,888 DEBUG   SenderThread:19352 [sender.py:send():378] send: stats
2024-03-15 22:46:11,352 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:46:11,941 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:11,941 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:11,941 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:11,941 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:12,837 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:14,813 ERROR   gpu       :19352 [interfaces.py:aggregate():161] Failed to serialize metric: division by zero
2024-03-15 22:46:14,814 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:14,814 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:14,903 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:16,450 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:46:16,824 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:16,824 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:16,824 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:16,824 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:16,964 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:17,275 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-15 22:46:17,290 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: stop_status
2024-03-15 22:46:17,290 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: stop_status
2024-03-15 22:46:18,837 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:18,837 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:18,837 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:18,837 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:19,015 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:20,064 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:20,848 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:20,848 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:20,848 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:20,848 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:21,470 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:46:22,110 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:22,854 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:22,854 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:22,854 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:22,854 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:24,169 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:24,860 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:24,860 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:24,860 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:24,860 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:26,222 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:26,555 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:46:26,864 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:26,864 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:26,864 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:26,864 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:28,277 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:28,866 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:28,866 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:28,866 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:28,866 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:30,367 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:30,869 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:30,869 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:30,869 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:30,869 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:31,587 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:46:32,285 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-15 22:46:32,285 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: stop_status
2024-03-15 22:46:32,286 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: stop_status
2024-03-15 22:46:32,424 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:32,871 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:32,871 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:32,871 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:32,871 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:34,471 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:34,881 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:34,882 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:34,882 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:34,882 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:36,523 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:36,619 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:46:36,885 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:36,885 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:36,885 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:36,885 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:38,565 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:38,889 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:38,889 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:38,889 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:38,889 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:40,619 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:40,894 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:40,895 DEBUG   SenderThread:19352 [sender.py:send():378] send: stats
2024-03-15 22:46:40,895 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:40,895 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:40,895 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:41,641 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:46:42,670 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:42,899 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:42,899 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:42,899 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:42,899 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:44,715 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:45,773 ERROR   gpu       :19352 [interfaces.py:aggregate():161] Failed to serialize metric: division by zero
2024-03-15 22:46:45,774 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:45,774 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:46,672 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:46:46,777 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:47,291 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-15 22:46:47,292 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: stop_status
2024-03-15 22:46:47,292 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: stop_status
2024-03-15 22:46:47,776 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:47,776 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:47,776 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:47,776 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:48,824 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:49,783 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:49,783 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:49,783 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:49,783 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:50,883 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:51,705 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:46:51,787 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:51,787 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:51,787 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:51,787 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:52,934 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:53,790 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:53,790 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:53,790 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:53,790 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:54,981 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:55,791 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:55,791 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:55,791 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:55,791 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:56,793 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:46:57,026 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:57,795 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:57,795 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:57,796 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:57,796 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:59,128 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:46:59,810 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:59,810 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:46:59,810 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:46:59,812 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:01,192 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:01,827 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:01,827 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:01,827 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:01,827 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:01,852 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:47:02,298 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-15 22:47:02,314 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: stop_status
2024-03-15 22:47:02,314 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: stop_status
2024-03-15 22:47:03,253 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:03,829 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:03,829 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:03,829 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:03,829 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:05,299 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:05,838 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:05,838 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:05,838 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:05,838 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:06,307 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:06,911 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:47:07,850 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:07,850 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:07,850 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:07,850 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:08,351 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:09,863 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:09,863 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:09,863 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:09,863 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:10,394 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:10,897 DEBUG   SenderThread:19352 [sender.py:send():378] send: stats
2024-03-15 22:47:11,870 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:11,870 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:11,870 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:11,870 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:11,935 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:47:12,456 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:13,881 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:13,881 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:13,881 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:13,881 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:14,527 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:16,581 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:16,753 ERROR   gpu       :19352 [interfaces.py:aggregate():161] Failed to serialize metric: division by zero
2024-03-15 22:47:16,755 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:16,755 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:17,010 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:47:17,303 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-15 22:47:17,318 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: stop_status
2024-03-15 22:47:17,318 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: stop_status
2024-03-15 22:47:18,639 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:18,755 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:18,756 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:18,756 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:18,756 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:20,674 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:20,768 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:20,768 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:20,768 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:20,768 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:22,058 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:47:22,731 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:22,777 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:22,777 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:22,777 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:22,777 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:24,785 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:24,785 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:24,785 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:24,785 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:24,786 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:26,797 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:26,797 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:26,797 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:26,797 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:26,830 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:27,186 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:47:28,799 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:28,799 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:28,799 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:28,799 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:28,878 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:30,801 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:30,801 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:30,801 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:30,801 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:30,928 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:32,228 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:47:32,306 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-15 22:47:32,336 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: stop_status
2024-03-15 22:47:32,337 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: stop_status
2024-03-15 22:47:32,806 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:32,806 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:32,806 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:32,806 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:32,996 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:34,810 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:34,810 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:34,810 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:34,810 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:35,063 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:36,811 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:36,811 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:36,811 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:36,811 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:37,138 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:37,230 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:47:38,822 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:38,822 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:38,822 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:38,822 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:39,203 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:40,831 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:40,831 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:40,831 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:40,831 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:40,910 DEBUG   SenderThread:19352 [sender.py:send():378] send: stats
2024-03-15 22:47:41,255 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:42,255 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:47:42,841 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:42,841 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:42,841 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:42,841 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:43,325 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:44,855 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:44,855 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:44,855 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:44,855 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:45,372 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:47,307 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:47:47,310 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-15 22:47:47,342 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: stop_status
2024-03-15 22:47:47,342 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: stop_status
2024-03-15 22:47:47,419 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:47,725 ERROR   gpu       :19352 [interfaces.py:aggregate():161] Failed to serialize metric: division by zero
2024-03-15 22:47:47,727 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:47,727 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:48,470 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:49,730 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:49,731 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:49,731 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:49,731 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:50,528 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:51,735 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:51,736 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:51,736 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:51,736 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:52,388 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:47:52,575 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:53,738 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:53,738 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:53,738 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:53,738 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:54,635 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:55,744 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:55,745 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:55,745 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:55,745 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:56,689 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:57,431 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:47:57,751 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:57,752 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:57,752 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:57,752 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:58,758 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:47:59,760 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:59,760 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:47:59,760 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:47:59,761 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:00,828 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:01,762 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:01,762 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:01,762 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:01,762 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:02,311 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-15 22:48:02,343 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: stop_status
2024-03-15 22:48:02,344 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: stop_status
2024-03-15 22:48:02,532 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:48:02,878 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:03,764 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:03,764 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:03,764 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:03,764 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:04,930 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:05,773 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:05,773 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:05,773 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:05,773 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:06,980 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:07,585 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:48:07,777 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:07,777 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:07,778 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:07,778 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:09,031 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:09,783 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:09,783 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:09,783 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:09,783 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:10,919 DEBUG   SenderThread:19352 [sender.py:send():378] send: stats
2024-03-15 22:48:11,087 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:11,784 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:11,784 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:11,784 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:11,784 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:12,590 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:48:13,133 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:13,791 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:13,791 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:13,791 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:13,791 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:15,186 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:15,803 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:15,803 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:15,803 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:15,803 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:17,258 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:17,317 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-15 22:48:17,334 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: stop_status
2024-03-15 22:48:17,335 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: stop_status
2024-03-15 22:48:17,614 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:48:18,688 ERROR   gpu       :19352 [interfaces.py:aggregate():161] Failed to serialize metric: division by zero
2024-03-15 22:48:18,690 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:18,690 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:19,305 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:20,701 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:20,701 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:20,701 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:20,701 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:21,350 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:22,630 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:48:22,703 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:22,703 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:22,703 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:22,703 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:23,425 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:24,717 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:24,718 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:24,718 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:24,718 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:25,475 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:26,732 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:26,732 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:26,732 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:26,732 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:27,529 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:27,667 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:48:28,746 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:28,746 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:28,746 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:28,746 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:29,581 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:30,751 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:30,751 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:30,751 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:30,751 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:31,626 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:32,330 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-15 22:48:32,345 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: stop_status
2024-03-15 22:48:32,345 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: stop_status
2024-03-15 22:48:32,692 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:48:32,762 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:32,762 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:32,762 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:32,762 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:33,699 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:34,775 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:34,775 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:34,775 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:34,775 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:35,765 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:36,780 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:36,780 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:36,781 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:36,781 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:37,816 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:48:37,816 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:38,785 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:38,786 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:38,786 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:38,786 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:39,872 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:40,795 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:40,795 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:40,795 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:40,795 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:40,920 DEBUG   SenderThread:19352 [sender.py:send():378] send: stats
2024-03-15 22:48:41,931 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:42,809 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:42,809 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:42,809 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:42,809 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:42,917 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:48:43,993 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:44,814 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:44,814 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:44,814 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:44,814 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:45,005 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:46,821 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:46,821 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:46,821 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:46,821 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:47,057 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:47,340 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: stop_status
2024-03-15 22:48:47,340 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-15 22:48:47,341 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: stop_status
2024-03-15 22:48:47,919 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:48:49,099 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:49,694 ERROR   gpu       :19352 [interfaces.py:aggregate():161] Failed to serialize metric: division by zero
2024-03-15 22:48:49,696 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:49,696 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:51,147 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:51,703 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:51,703 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:51,703 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:51,703 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:52,958 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:48:53,198 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:53,711 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:53,711 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:53,711 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:53,711 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:55,230 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:55,716 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:55,716 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:55,716 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:55,716 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:57,296 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:57,730 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:57,731 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:57,731 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:57,731 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:57,968 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:48:59,348 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:48:59,734 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:59,734 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:48:59,734 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:48:59,734 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:49:01,399 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:49:01,747 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:49:01,747 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:49:01,747 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:49:01,747 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:49:02,341 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-15 22:49:02,356 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: stop_status
2024-03-15 22:49:02,357 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: stop_status
2024-03-15 22:49:02,990 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:49:03,464 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:49:03,746 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:49:03,746 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:49:03,746 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:49:03,746 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:49:05,522 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:49:05,753 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:49:05,753 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:49:05,753 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:49:05,753 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:49:07,585 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:49:07,757 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:49:07,757 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:49:07,757 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:49:07,757 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:49:07,993 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:49:09,634 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:49:09,763 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:49:09,763 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Unknown Error
2024-03-15 22:49:09,763 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:49:09,763 ERROR   gpu       :19352 [interfaces.py:monitor():144] Failed to sample metric: Not Supported
2024-03-15 22:49:10,922 DEBUG   SenderThread:19352 [sender.py:send():378] send: exit
2024-03-15 22:49:10,923 INFO    SenderThread:19352 [sender.py:send_exit():585] handling exit code: 1
2024-03-15 22:49:10,923 INFO    SenderThread:19352 [sender.py:send_exit():587] handling runtime: 271
2024-03-15 22:49:10,935 INFO    SenderThread:19352 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-15 22:49:10,935 INFO    SenderThread:19352 [sender.py:send_exit():593] send defer
2024-03-15 22:49:10,935 DEBUG   SenderThread:19352 [sender.py:send():378] send: stats
2024-03-15 22:49:10,935 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: defer
2024-03-15 22:49:10,935 INFO    HandlerThread:19352 [handler.py:handle_request_defer():172] handle defer: 0
2024-03-15 22:49:10,935 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: defer
2024-03-15 22:49:10,935 INFO    SenderThread:19352 [sender.py:send_request_defer():609] handle sender defer: 0
2024-03-15 22:49:10,935 INFO    SenderThread:19352 [sender.py:transition_state():613] send defer: 1
2024-03-15 22:49:10,935 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: defer
2024-03-15 22:49:10,935 INFO    HandlerThread:19352 [handler.py:handle_request_defer():172] handle defer: 1
2024-03-15 22:49:10,935 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: defer
2024-03-15 22:49:10,935 INFO    SenderThread:19352 [sender.py:send_request_defer():609] handle sender defer: 1
2024-03-15 22:49:10,935 INFO    SenderThread:19352 [sender.py:transition_state():613] send defer: 2
2024-03-15 22:49:10,935 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: defer
2024-03-15 22:49:10,935 INFO    HandlerThread:19352 [handler.py:handle_request_defer():172] handle defer: 2
2024-03-15 22:49:10,935 INFO    HandlerThread:19352 [system_monitor.py:finish():203] Stopping system monitor
2024-03-15 22:49:10,947 DEBUG   SystemMonitor:19352 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-03-15 22:49:10,947 DEBUG   SystemMonitor:19352 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-03-15 22:49:10,947 INFO    HandlerThread:19352 [interfaces.py:finish():202] Joined cpu monitor
2024-03-15 22:49:10,948 INFO    HandlerThread:19352 [interfaces.py:finish():202] Joined disk monitor
2024-03-15 22:49:11,688 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:49:11,688 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_created():271] file/dir created: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\wandb-summary.json
2024-03-15 22:49:11,808 ERROR   gpu       :19352 [interfaces.py:aggregate():161] Failed to serialize metric: division by zero
2024-03-15 22:49:11,808 INFO    HandlerThread:19352 [interfaces.py:finish():202] Joined gpu monitor
2024-03-15 22:49:11,808 INFO    HandlerThread:19352 [interfaces.py:finish():202] Joined memory monitor
2024-03-15 22:49:11,808 INFO    HandlerThread:19352 [interfaces.py:finish():202] Joined network monitor
2024-03-15 22:49:11,809 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: defer
2024-03-15 22:49:11,809 INFO    SenderThread:19352 [sender.py:send_request_defer():609] handle sender defer: 2
2024-03-15 22:49:11,809 INFO    SenderThread:19352 [sender.py:transition_state():613] send defer: 3
2024-03-15 22:49:11,809 DEBUG   SenderThread:19352 [sender.py:send():378] send: stats
2024-03-15 22:49:11,809 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: defer
2024-03-15 22:49:11,809 INFO    HandlerThread:19352 [handler.py:handle_request_defer():172] handle defer: 3
2024-03-15 22:49:11,810 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: defer
2024-03-15 22:49:11,810 INFO    SenderThread:19352 [sender.py:send_request_defer():609] handle sender defer: 3
2024-03-15 22:49:11,810 INFO    SenderThread:19352 [sender.py:transition_state():613] send defer: 4
2024-03-15 22:49:11,810 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: defer
2024-03-15 22:49:11,810 INFO    HandlerThread:19352 [handler.py:handle_request_defer():172] handle defer: 4
2024-03-15 22:49:11,810 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: defer
2024-03-15 22:49:11,810 INFO    SenderThread:19352 [sender.py:send_request_defer():609] handle sender defer: 4
2024-03-15 22:49:11,810 INFO    SenderThread:19352 [sender.py:transition_state():613] send defer: 5
2024-03-15 22:49:11,810 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: defer
2024-03-15 22:49:11,810 INFO    HandlerThread:19352 [handler.py:handle_request_defer():172] handle defer: 5
2024-03-15 22:49:11,812 DEBUG   SenderThread:19352 [sender.py:send():378] send: summary
2024-03-15 22:49:11,813 INFO    SenderThread:19352 [sender.py:_save_file():1365] saving file wandb-summary.json with policy end
2024-03-15 22:49:11,813 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: defer
2024-03-15 22:49:11,813 INFO    SenderThread:19352 [sender.py:send_request_defer():609] handle sender defer: 5
2024-03-15 22:49:11,813 INFO    SenderThread:19352 [sender.py:transition_state():613] send defer: 6
2024-03-15 22:49:11,813 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: defer
2024-03-15 22:49:11,813 INFO    HandlerThread:19352 [handler.py:handle_request_defer():172] handle defer: 6
2024-03-15 22:49:11,814 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: defer
2024-03-15 22:49:11,814 INFO    SenderThread:19352 [sender.py:send_request_defer():609] handle sender defer: 6
2024-03-15 22:49:11,814 INFO    SenderThread:19352 [sender.py:transition_state():613] send defer: 7
2024-03-15 22:49:11,814 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:49:11,814 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: defer
2024-03-15 22:49:11,814 INFO    HandlerThread:19352 [handler.py:handle_request_defer():172] handle defer: 7
2024-03-15 22:49:11,815 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: defer
2024-03-15 22:49:11,815 INFO    SenderThread:19352 [sender.py:send_request_defer():609] handle sender defer: 7
2024-03-15 22:49:11,938 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: poll_exit
2024-03-15 22:49:12,703 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\wandb-summary.json
2024-03-15 22:49:13,139 INFO    SenderThread:19352 [sender.py:transition_state():613] send defer: 8
2024-03-15 22:49:13,139 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: poll_exit
2024-03-15 22:49:13,139 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: defer
2024-03-15 22:49:13,140 INFO    HandlerThread:19352 [handler.py:handle_request_defer():172] handle defer: 8
2024-03-15 22:49:13,140 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: defer
2024-03-15 22:49:13,140 INFO    SenderThread:19352 [sender.py:send_request_defer():609] handle sender defer: 8
2024-03-15 22:49:13,140 INFO    SenderThread:19352 [job_builder.py:build():298] Attempting to build job artifact
2024-03-15 22:49:13,141 INFO    SenderThread:19352 [job_builder.py:_get_source_type():428] is repo sourced job
2024-03-15 22:49:13,250 INFO    SenderThread:19352 [job_builder.py:build():404] adding wandb-job metadata file
2024-03-15 22:49:13,258 INFO    SenderThread:19352 [sender.py:transition_state():613] send defer: 9
2024-03-15 22:49:13,259 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: defer
2024-03-15 22:49:13,259 INFO    HandlerThread:19352 [handler.py:handle_request_defer():172] handle defer: 9
2024-03-15 22:49:13,259 DEBUG   SenderThread:19352 [sender.py:send():378] send: artifact
2024-03-15 22:49:13,716 INFO    Thread-16 :19352 [dir_watcher.py:_on_file_modified():288] file/dir modified: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:49:13,964 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: poll_exit
2024-03-15 22:49:14,481 INFO    wandb-upload_0:19352 [upload_job.py:push():86] Skipped uploading C:\Users\USER\AppData\Local\wandb\wandb\artifacts\staging\tmp_i51u831
2024-03-15 22:49:14,826 INFO    wandb-upload_1:19352 [upload_job.py:push():89] Uploaded file C:\Users\USER\AppData\Local\wandb\wandb\artifacts\staging\tmp7cmf8fft
2024-03-15 22:49:16,050 INFO    SenderThread:19352 [sender.py:send_artifact():1443] sent artifact job-https___github.com_singforai_mini-marl.git_vdn_main.py - {'id': 'QXJ0aWZhY3Q6NzU3MDkyNDcz', 'state': 'PENDING', 'artifactSequence': {'id': 'QXJ0aWZhY3RDb2xsZWN0aW9uOjE0NjY3ODI4OQ==', 'latestArtifact': {'id': 'QXJ0aWZhY3Q6NzU3MDY4NDc0', 'versionIndex': 9}}}
2024-03-15 22:49:16,050 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: defer
2024-03-15 22:49:16,050 INFO    SenderThread:19352 [sender.py:send_request_defer():609] handle sender defer: 9
2024-03-15 22:49:16,050 INFO    SenderThread:19352 [dir_watcher.py:finish():358] shutting down directory watcher
2024-03-15 22:49:16,739 INFO    SenderThread:19352 [dir_watcher.py:finish():388] scan: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files
2024-03-15 22:49:16,739 INFO    SenderThread:19352 [dir_watcher.py:finish():402] scan save: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\conda-environment.yaml conda-environment.yaml
2024-03-15 22:49:16,740 INFO    SenderThread:19352 [dir_watcher.py:finish():402] scan save: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\config.yaml config.yaml
2024-03-15 22:49:16,742 INFO    SenderThread:19352 [dir_watcher.py:finish():402] scan save: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log output.log
2024-03-15 22:49:16,746 INFO    SenderThread:19352 [dir_watcher.py:finish():402] scan save: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\requirements.txt requirements.txt
2024-03-15 22:49:16,748 INFO    SenderThread:19352 [dir_watcher.py:finish():402] scan save: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\wandb-metadata.json wandb-metadata.json
2024-03-15 22:49:16,748 INFO    SenderThread:19352 [dir_watcher.py:finish():402] scan save: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\wandb-summary.json wandb-summary.json
2024-03-15 22:49:16,753 INFO    SenderThread:19352 [sender.py:transition_state():613] send defer: 10
2024-03-15 22:49:16,753 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: poll_exit
2024-03-15 22:49:16,754 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: defer
2024-03-15 22:49:16,754 INFO    HandlerThread:19352 [handler.py:handle_request_defer():172] handle defer: 10
2024-03-15 22:49:16,754 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: defer
2024-03-15 22:49:16,754 INFO    SenderThread:19352 [sender.py:send_request_defer():609] handle sender defer: 10
2024-03-15 22:49:16,754 INFO    SenderThread:19352 [file_pusher.py:finish():172] shutting down file pusher
2024-03-15 22:49:16,996 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: poll_exit
2024-03-15 22:49:16,996 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: poll_exit
2024-03-15 22:49:17,201 INFO    wandb-upload_0:19352 [upload_job.py:push():131] Uploaded file C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\conda-environment.yaml
2024-03-15 22:49:17,327 INFO    wandb-upload_3:19352 [upload_job.py:push():131] Uploaded file C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\requirements.txt
2024-03-15 22:49:17,351 INFO    wandb-upload_1:19352 [upload_job.py:push():131] Uploaded file C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\config.yaml
2024-03-15 22:49:17,371 INFO    wandb-upload_2:19352 [upload_job.py:push():131] Uploaded file C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\output.log
2024-03-15 22:49:17,446 INFO    wandb-upload_4:19352 [upload_job.py:push():131] Uploaded file C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\files\wandb-summary.json
2024-03-15 22:49:17,660 INFO    Thread-15 :19352 [sender.py:transition_state():613] send defer: 11
2024-03-15 22:49:17,661 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: defer
2024-03-15 22:49:17,661 INFO    HandlerThread:19352 [handler.py:handle_request_defer():172] handle defer: 11
2024-03-15 22:49:17,661 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: defer
2024-03-15 22:49:17,661 INFO    SenderThread:19352 [sender.py:send_request_defer():609] handle sender defer: 11
2024-03-15 22:49:17,661 INFO    SenderThread:19352 [file_pusher.py:join():178] waiting for file pusher
2024-03-15 22:49:17,661 INFO    SenderThread:19352 [sender.py:transition_state():613] send defer: 12
2024-03-15 22:49:17,661 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: defer
2024-03-15 22:49:17,661 INFO    HandlerThread:19352 [handler.py:handle_request_defer():172] handle defer: 12
2024-03-15 22:49:17,661 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: defer
2024-03-15 22:49:17,661 INFO    SenderThread:19352 [sender.py:send_request_defer():609] handle sender defer: 12
2024-03-15 22:49:17,661 INFO    SenderThread:19352 [file_stream.py:finish():595] file stream finish called
2024-03-15 22:49:17,997 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: poll_exit
2024-03-15 22:49:18,218 INFO    SenderThread:19352 [file_stream.py:finish():599] file stream finish is done
2024-03-15 22:49:18,218 INFO    SenderThread:19352 [sender.py:transition_state():613] send defer: 13
2024-03-15 22:49:18,219 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: poll_exit
2024-03-15 22:49:18,219 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: defer
2024-03-15 22:49:18,219 INFO    HandlerThread:19352 [handler.py:handle_request_defer():172] handle defer: 13
2024-03-15 22:49:18,219 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: status_report
2024-03-15 22:49:18,220 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: defer
2024-03-15 22:49:18,220 INFO    SenderThread:19352 [sender.py:send_request_defer():609] handle sender defer: 13
2024-03-15 22:49:18,220 INFO    SenderThread:19352 [sender.py:transition_state():613] send defer: 14
2024-03-15 22:49:18,220 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: defer
2024-03-15 22:49:18,220 DEBUG   SenderThread:19352 [sender.py:send():378] send: final
2024-03-15 22:49:18,220 INFO    HandlerThread:19352 [handler.py:handle_request_defer():172] handle defer: 14
2024-03-15 22:49:18,220 DEBUG   SenderThread:19352 [sender.py:send():378] send: footer
2024-03-15 22:49:18,221 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: defer
2024-03-15 22:49:18,221 INFO    SenderThread:19352 [sender.py:send_request_defer():609] handle sender defer: 14
2024-03-15 22:49:18,221 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: poll_exit
2024-03-15 22:49:18,221 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: poll_exit
2024-03-15 22:49:18,221 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: server_info
2024-03-15 22:49:18,222 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: poll_exit
2024-03-15 22:49:18,222 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: get_summary
2024-03-15 22:49:18,222 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: poll_exit
2024-03-15 22:49:18,222 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: sampled_history
2024-03-15 22:49:18,222 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: server_info
2024-03-15 22:49:18,222 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: internal_messages
2024-03-15 22:49:18,224 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: job_info
2024-03-15 22:49:18,397 DEBUG   SenderThread:19352 [sender.py:send_request():405] send_request: job_info
2024-03-15 22:49:18,397 INFO    MainThread:19352 [wandb_run.py:_footer_history_summary_info():3851] rendering history
2024-03-15 22:49:18,397 INFO    MainThread:19352 [wandb_run.py:_footer_history_summary_info():3883] rendering summary
2024-03-15 22:49:18,397 INFO    MainThread:19352 [wandb_run.py:_footer_sync_info():3810] logging synced files
2024-03-15 22:49:18,399 DEBUG   HandlerThread:19352 [handler.py:handle_request():146] handle_request: shutdown
2024-03-15 22:49:18,399 INFO    HandlerThread:19352 [handler.py:finish():869] shutting down handler
2024-03-15 22:49:19,227 INFO    WriterThread:19352 [datastore.py:close():296] close: C:\Users\USER\Desktop\mini-marl\vdn\wandb\run-20240315_224439-0w0d7rkn\run-0w0d7rkn.wandb
2024-03-15 22:49:19,399 INFO    SenderThread:19352 [sender.py:finish():1521] shutting down sender
2024-03-15 22:49:19,399 INFO    SenderThread:19352 [file_pusher.py:finish():172] shutting down file pusher
2024-03-15 22:49:19,399 INFO    SenderThread:19352 [file_pusher.py:join():178] waiting for file pusher
